docs/fuse-box/tutorials/api-trade-quotes.mdx

```diff
@@ -0,0 +1,334 @@
+
+## Prerequisites
+Before starting, ensure you have the following:
+
+Node.js installed on your machine.
+- Code Editor: Use your preferred code editor; VS Code is recommended.
+- An EOA wallet with a private key. You can use an existing one or create a new wallet.
+- A basic understanding of React.js and Next.js.
+- An API key from the Fuse Console. Get one [here](https://console.fuse.io/build).
+
+## Step 1: Set Up a Next.js Project
+If you haven't already set up a Next.js project, you can create one using the following commands:
+
+```bash
+npx create-next-app my-project
+cd my-project
+```
+
+Answer the required prompts from NextJS in the terminal. Note that we use TypeScript and Tailwind CSS for this application.
+
+## Step 2: API Endpoint
+In this example tutorial, we are going to build a Form element that takes in Tokens for BUY and SELL orders and an amount and returns the response in a table. We will use the Axios library to call the Fuse REST APIs and return the responses.
+
+The endpoint that we will use in this tutorial is the Get List of Tokens Owned by Address.
+
+The base URL is https://api.fuse.io/api/v1/trade/quote, and it takes three required parameters: **buyToken**, **sellToken**, and **amount**. You can get an API key from [Console](https://console.fuse.io/build).
+
+The URL, when appended with the parameters, for example:
+
+https://api.fuse.io/api/v1/trade/quote?apiKey=YOUR_API_KEY&sellToken=FUSE&buyToken=0x34Ef2Cc892a88415e9f02b91BfA9c91fC0bE6bD4&sellAmount=1000000000000000000
+## Step 2: Install Axios
+Axios is a promise-based HTTP client for the browser and Node.js. Install Axios to handle the API requests:
+```bash
+npm install axios
+```
+
+## Step 3: Fetch the API response.
+
+In this step, we will test calling the API and parsing the response to the web browser console. Add the following code to the `index.tsx` file before the return statement.
+
+```javascript
+const apiUrl = `https://api.fuse.io/api/v1/trade/quote?apiKey=YOUR_API_KEY&sellToken=FUSE&buyToken=0x34Ef2Cc892a88415e9f02b91BfA9c91fC0bE6bD4&sellAmount=1000000000000000000`
+
+async function fetchData() {
+    try {
+      const response = await fetch(apiUrl);
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const data = await response.json();
+      console.log(data);
+      } catch (error) {
+      console.error('There was a problem with the fetch operation:', error);
+    }
+  }
+
+fetchData();
+```
+
+Save the file.
+
+Run the application using the command:
+
+```bash
+npm run dev
+```
+Open your browser, go to `http://localhost:3000`, and open the developer console. You will find the results logged to the console.
```

Comment:
Didnt saw any logs in browser console, but saw it in terminal of project.

Reply:
<img width="561" alt="Screenshot 2024-06-01 at 20 39 05" src="https://github.com/fuseio/fuse-docs/assets/10050194/f735c279-e1e2-4d96-928e-be8de0b9d08c">

The response is logged to the Browser Console. See line 65.

Reply:
Saw it now, after i rebuildproject with configuration you provided.